src/load/azext_load/data_plane/load_test/help.py

```diff
@@ -0,0 +1,137 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+helps = {}
+
+helps[
+    "load test create"
+] = """
+type: command
+short-summary: Create a new load test.
+long-summary: Create a new load test.
+"""
+
+helps[
+    "load test list"
+] = """
+type: command
+short-summary: List all tests.
+long-summary: List all tests in the given load test resource.
+"""
+
+helps[
+    "load test show"
+] = """
+type: command
+short-summary: Show details of test.
+long-summary: Show details of the test identified by given test-id.
+"""
+
+helps[
+    "load test update"
+] = """
+type: command
+short-summary: Update an existing load test.
+long-summary: Update an existing load test.
```

Comment:
`long-summary` is used to add more description for the command. If it's almost the same as `short-summary` you should remove the `long-summary`

Reply:
Updated help as suggested